README.md

```diff
@@ -130,3 +130,62 @@ To debug native code with VSCode or CLion or any other IDE:
 3. Set the parameters to be the ones used by the ```mvn``` script, as per above
 4. Set the working directory to the `aws-crt-java` directory
 5. On windows, you will need to manually load the PDB via the Modules window in Visual Studio, as it is not embedded in the JAR. It will be in the ```target/cmake-build/lib/windows/<arch>``` folder.
+
+## Platform-Specific JARs
+
+Since v0.20.5, platform-specific JARs are available to reduce the size of the artificts. To consume the platform-specific JAR of aws-crt-java, you can use Maven classifier (https://maven.apache.org/pom.html) property to specify the platform. Without classifier, you will get the uber jar to support all distributions.
+
+``` xml
+        <!-- Linux x86_64 distribution  -->
+        <dependency>
+            <groupId>software.amazon.awssdk.crt</groupId>
+            <artifactId>aws-crt</artifactId>
+            <version>0.20.5</version>
+            <classifier>linux-x86_64</classifier>
+        </dependency>
+
```

Comment:
Make these two different code blocks. Someone would never want to specify both the uber jar AND a platform specific jar, but since they're the same code block here it kind of looks like that's what you're supposed to do